.github/workflows/build.yml

```diff
@@ -70,7 +96,7 @@ jobs:
         body_path: "${{ steps.relnotes.outputs.file }}"
         
   build:
-    needs: [calc_ver]
+    needs: [calc_ver, tests]
```

Comment:
> In certain cases yes, but here the aim is not to build if tests are failing.

I don't like this approach because compiling for testing is long.  Lets do build and tests in parallel. The dependency of Release on tests will be enough.